src/cursor/cur_file.c

```diff
@@ -412,8 +412,11 @@ __curfile_remove(WT_CURSOR *cursor)
     __wt_stat_usecs_hist_incr_opwrite(session, WT_CLOCKDIFF_US(time_stop, time_start));
 
     /* If we've lost an initial position, we must fail. */
-    if (positioned && !F_ISSET(cursor, WT_CURSTD_KEY_INT))
+    if (positioned && !F_ISSET(cursor, WT_CURSTD_KEY_INT)) {
+        WT_IGNORE_RET(__wt_msg(
+          session, "WT_ROLLBACK: rolling back cursor remove as initial position was lost"));
         WT_ERR(WT_ROLLBACK);
+    }
```

Comment:
I added the log message here and in txn_ckpt because the rollback reason is not saved in these places, and I thought it might be more informative to provide specific reasons for the WT_ROLLBACK error. Let me know if you think otherwise

Reply:
I'm only questioning why we want to log a message. If there's a reason, I'm on-board.